.github/ISSUE_TEMPLATE/add-project.md

```diff
@@ -1,22 +1,27 @@
 ---
 name: Add a Project
-about: Add Smart Contract
+about: Add a new Web3 Project
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-### Description 
-Describe more about the issue.
+
+### Project title
+(Write project title over here)
+
+### Project Description
+(Describe the project over here)
 
 ### Doc Domain
 <!--
 Leave it unchecked Owner will mark it :)
 -->
-- [ ] Solidity
-- [ ] Python
-- [ ] JavaScript
+- [] Solidity , Html, Css , JavaScript , Web3.js /Ether.js 
```

Comment:
remove line 21 , 22 and 23 and add 
the lines which i  mentioned in the improvements.md file